cmd/zhack/zhack.c

```diff
@@ -580,7 +583,7 @@ zhack_do_scrub(int argc, char **argv)
 	zfs_no_scrub_prefetch = B_TRUE;
 
 
-	while ((c = getopt(argc, argv, "D:FPTp:rv")) != -1) {
+	while ((c = getopt(argc, argv, "D:PRTnrv")) != -1) {
 		switch (c) {
 		case 'D':
 			// How much of the DDT are we scanning?
```

Comment:
One more style issue, please don't use C++ style comments `//` you should use `/* ... */`.